PurchaseController/Classes/Storage.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Storage.swift
+//  Pods
+//
+//  Copyright (c) 2019 dashdevs.com. All rights reserved.
+//
+
+import Foundation
+
+/// Class to store all data, that should be accessible from one point
+class Storage {
+    
+    private var persistor: PurchasePersistor
+    public private(set) var sessionReceipt: Receipt?
+    
+    /// Defult init with persistor object
+    ///
+    /// - Parameter persistor: object conforms PurchasePersistor protocol
+    init(persistor: PurchasePersistor) {
+        self.persistor = PurchasePersistorImplementation()
+    }
+}
+
+// MARK: - Access
+
+extension Storage {
+    /// Method to set session receipt instance
```

Comment:
Please update code documentation to a new markdown style, i.e.
```
/**
 Documentation
*/

Reply:
done